PeerioTechnologies/peerio-desktop
# fix: set NODE_ENV in renderer process too

On Linux, renderer processes don't seem to inherit environment variables of the main process.

Thus, renderer processes think they're in the dev environment. Which makes `console.history` not work.

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/8754/desktop-logs-to-clipboard-for-linux 

#### Testing instructions  

* Open console and verify that it doesn't say "DEV environment detected".
* Open Settings > Help and copy logs; verify that they were copied indeed.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
